Challenge02/flor.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+<head>
+    <title>Metal Music Blog</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="../css/flor.css" type="text/css" />

```

Comment:
I notice that at this point you did not add your .css file. In that case, line 8 does not have much sense if the file does not exist